### DISCUSSION TITLE
Github asking me another email when reapplying which i don't have access to

### DISCUSSION THREAD

**Discussion by anis-18** [community/community#57450] [GitHub Education]
### Hi everyone,

Hello, I received an email informing me that i have to reverify my account in order to continue using fritiha student pack or my account will be downgraded by may 30th. Please note that i've applied for github student pack in 6th february.
Thé problem is that when sending a new request i'm surprised that github is asking me to enter a specific email to which i don't have access.
We require applicants of Kazan National Research Technological University (Казанский национальный исследовательский технологический университет) to use one of these school-issued email addresses to apply:

knitukazan.onmicrosoft.com
Please select a different email address.
The problem is that as students in this university we're not provided and WE don't have access to these kind of emails.
WE have a personal profile account and a personal profile page in the university website.
In february 6th when i applied there was not such an email.
Please Can You help me, how Can i resolve this problem ?
Cheers

**Comment by samus-aran** [comment]
Hi @anisamirouche 

First off I want to thank you for sharing your experiences and that we absolutely understand your frustration. We are trying our best to continually improve our system to be as easy as possible for legitimate students everywhere to verify their academic status, while also ensuring that bad actors impersonating students don't jeopardise our ability to provide academic benefits of value.
As you can imagine there are many types of academic verification and our system is constantly evolving to accommodate this without setting the bar too high. However sometimes we may require a little more evidence from the student applying and we thank you for being patient. In this case it would be the Academic email(s) which has been associated with this school.
We would recommend to you speak to your institution regarding to acquiring an email address to proceed.
